README.md

```diff
@@ -1 +1,283 @@
+
+# 📖 [Hello_World] <a name="about-project"></a>
+
+This project is about Microverse Linters configuration and ensuring that all linters checks pass, and also add & display a simple HTML text and basic CSS styling.  
+
+**[Hello_World]** is a simple massage for linter practice
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">Git Hub</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+<!-- <details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Linters Setup]**
+- **[Follow HTML and CSS best practice]**
+- **[Display Text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://tsheporamantso.github.io/hello-microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use git clone command or download ZIP folder 
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:tsheporamantso/Hello_World.git
+```
+
+
+### Install
+
+Install this project with:
+
+npm
+
+Example command:
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start or live server
+
+
+Example command:
+
+```sh
+  GitHub Pages Server
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+
+Example command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub Pages 
+
+
+Example: 
+
+```sh
+https://tsheporamantso.github.io/Hello_World/
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gladwin Tshepo Ramantso**
+
+- GitHub: [@tsheporamantso](https://github.com/tsheporamantso)
+- Twitter: [@ramgt001](https://twitter.com/ramgt001)
+- LinkedIn: [Tshepo Gladwin Ramantso](https://www.linkedin.com/in/tshepo-ramantso-b6a35433/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
```

Comment:
- [ ] [Optional] I would kindly suggest you add 1-3 future features that you will be having in this project in this section, the same way you add them in the key features. I marked this optional because it wasn't raised by the previous reviewer

Reply:
Noted, and thank you so much.